extensions/amazon-lambda/deployment/src/main/java/io/quarkus/amazon/lambda/deployment/AmazonLambdaProcessor.java

```diff
@@ -102,7 +102,7 @@ AdditionalApplicationArchiveMarkerBuildItem marker() {
 
             ClassInfo current = info;
             boolean done = false;
-            boolean streamHandler = false;
+            boolean streamHandler = info.superName().equals(SKILL_STREAM_HANDLER) ? true : false;
             while (current != null && !done) {
```

Comment:
Do we need to execute this if streamHandler is true?

Reply:
Discussed below.